packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -119,7 +119,7 @@ export interface ISemanticColors extends ISemanticTextColors {
    */
   defaultStateBackground: string;
 
-  //// Invariants - slots that rarely change color theme-to-theme because the color has meaning
+  //// Invariants - slots that rarely change color theme-to-theme because the color has meaning. Exclusive to List items.
```

Comment:
This isn't true, invariant slots can be used anywhere. errorText, for example, would be used across the entire product.